docs/index.md

```diff
@@ -19,6 +19,19 @@ This documentation conforms to [RFC 2119][], which says:
 
 [MSON][] is used throughout this document to define elements and structures.
 
+## Contents
+
+```eval_rst
```

Comment:
I'm pretty amazed the combo of heading/eval_rst works corretly for you. I remember I had some issues with it. But - good for you 😄